### PR TITLE
Feature/animations

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

**Comment by zenangst** [comment]
@vadymmarkov The tests don't run :disappointed:

**Comment by zenangst** [comment]
Seems like Travis is being a dick.

File: Pod/Demo/AppDelegate.swift [hunk 1]
```diff
@@ -104,6 +58,127 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+  func addBackgroundViews() {
+    let cloudView1 = UIImageView(image: UIImage(named: "cloud1"))
+    let backViewModel1 = BackViewModel(view: cloudView1,
+      position: Position(left: 0.7, top: 0.7))
+
+    tutorialController.addBackViewModels([backViewModel1])
+  }
+
+  func addAnimations() {
+    let image1 = UIImageView(image: UIImage(named: "cloud1"))
```
Comment (kostiakoval) [inline Pod/Demo/AppDelegate.swift]: Using map and PF could prove this code
let images = ["could1", "could2"].map { UIImageView(image: UIImage(named: $0)) }

**Comment by zenangst** [comment]
But the tests run locally meow.

File: Pod/Demo/AppDelegate.swift (hunk 1 above)
Comment (zenangst) [inline Pod/Demo/AppDelegate.swift]: <3

Comment (vadymmarkov) [inline Pod/Demo/AppDelegate.swift]: @kostiakoval Good point!

Comment (kostiakoval) [inline Pod/Demo/AppDelegate.swift]: and maybe similar for tutorialcontroller. with FP you can separate routine work (the same code) very easily

**Pull request merged**